.pre-commit-config.yaml

```diff
@@ -59,6 +59,13 @@ repos:
     - id: flake8
       args: ["--count", "--statistics"]
 
+-   repo: https://github.com/pre-commit/mirrors-mypy
+    rev: v0.961
+    hooks:
+      - id: mypy
+        files: src
+        args: []
```

Comment:
Uniformed question: Why is the `[]` needed for `args`? If it isn't can we drop it to avoid confusion in the future?

Reply:
next PR is going to flesh it out to fix up missing typehints for imports. It'll be needed.